add task solution,fixed last commits

SerjDSA opened the pull request:
[DEMO LINK](https://serjdsa.github.io/layout_landing-page/)

Andriy-Shymkiv reviewed (changes requested):
good job!
to improve:
- increase indent between title and picture a bit:
![image](https://github.com/mate-academy/layout_landing-page/assets/102289654/6b7a75cb-b5fc-4564-a220-29e347d96ad9)
- this inputs should be smaller I suppose:
![image](https://github.com/mate-academy/layout_landing-page/assets/102289654/8c8dec5a-5a41-4c51-b957-4c7b69a25070)
- link for google map looks different than the other ones:
![image](https://github.com/mate-academy/layout_landing-page/assets/102289654/8ecc0064-d20b-446a-9cd0-80c85ab584c0)

sTorba24 reviewed (changes requested):
1.Inputs should be smaller
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/e9475c59-e7de-4572-a644-b602df278df6)

2. remove autocomplete styles(blue background)
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/1f6bd109-f71e-4765-b8fe-b440d1adfd14)

3. links should have the same styles here, check mockup
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/1f6d2bca-6500-40fc-bcc4-b71885ff79d4)

SerjDSA reviewed (commented):
[DEMO LINK](https://serjdsa.github.io/layout_landing-page/)

SerjDSA reviewed (commented):
[DEMO LINK](https://serjdsa.github.io/layout_landing-page/)

Esceype reviewed (changes requested):
GJ!
Still need to fix:
- disable resize on textarea:
![image](https://github.com/mate-academy/layout_landing-page/assets/64101898/2d565816-f0e4-492b-88c1-11e424fdf3b9)
- scale should be smooth when mouse enter and when it leaves:
![image](https://github.com/mate-academy/layout_landing-page/assets/64101898/dfd913a6-f777-4ebb-8bf5-dec7cc72075b)
- this picture on tablet doesn't look like the one on the design:
Design:
![image](https://github.com/mate-academy/layout_landing-page/assets/64101898/1ebc8e10-e78a-4e55-a5d7-c08f8c4b5a5e)
Your:
![image](https://github.com/mate-academy/layout_landing-page/assets/64101898/ac10ddf2-f6a5-4878-b4e6-24ca5b511158)
- Start of the text on tablet should be the in same align as heading of section:
![image](https://github.com/mate-academy/layout_landing-page/assets/64101898/f87ce190-2f8a-4691-a7ce-8f174ef2fe2f)

nazarmatsevych reviewed (changes requested):
Looks like you didn't push any changes since last review.

If you have any questions - feel free to ask them in fe_chat

SerjDSA reviewed (commented):
[DEMO LINK](https://serjdsa.github.io/layout_landing-page/)

SerjDSA commented:
> Looks like you didn't push any changes since last review.
> 
> If you have any questions - feel free to ask them in fe_chat

Thanks for advise, but there are still possible solutions.

Andriy-Shymkiv reviewed (changes requested):
very good! the last thing to fix:
- still can change the size of textarea
![image](https://github.com/mate-academy/layout_landing-page/assets/102289654/604bf11c-6b6f-4aaf-b915-9d96057e6056)

SerjDSA commented:
> very good! the last thing to fix:
> 
> * still can change the size of textarea
>   ![image](https://user-images.githubusercontent.com/102289654/280517421-604bf11c-6b6f-4aaf-b915-9d96057e6056.png)



> very good! the last thing to fix:
> 
> * still can change the size of textarea
>   ![image](https://user-images.githubusercontent.com/102289654/280517421-604bf11c-6b6f-4aaf-b915-9d96057e6056.png)

Good day, Andriy. Google Chrome working without any problems, but will solve this problem.

SerjDSA reviewed (commented):
[DEMO LINK](https://serjdsa.github.io/layout_landing-page/)

Esceype reviewed (changes requested):
Please disable resize prop on textarea tag.

SanyaBratashchuk reviewed (changes requested):
1. add favicon

SerjDSA reviewed (commented):
[DEMO LINK](https://serjdsa.github.io/layout_landing-page/)

SerjDSA commented:
> 1. add favicon

favicon has name close.icon

TarasHoliuk reviewed (changes requested):
But favicon doesn't work
![image](https://github.com/mate-academy/layout_landing-page/assets/86923089/172a08ec-2c06-40ec-b1aa-ab2fe8e806bf)

Make sure it works (maybe redeploy your changes)

SerjDSA commented:
> > 1. add favicon
> 
>

if it need I will do

SerjDSA reviewed (commented):
[DEMO LINK](https://serjdsa.github.io/layout_landing-page/)
